users/models.py

```diff
@@ -0,0 +1,29 @@
+from django.db import models
+
+# Create your models here.
+from django.contrib.auth.models import User
+from django.db.models.signals import pre_save, post_save
+from django.dispatch import receiver
+from django.contrib.auth.models import Permission
+
+
+
+
+
+@receiver(pre_save, sender=User)
+def set_group(sender, **kwargs):
+    user = kwargs["instance"]
+    if user.is_staff == 0:
+        user.is_staff = 1
+        user.save()
+    else:
+        pass
+
+@receiver(post_save, sender=User)
+def set_permissions(sender, **kwargs):
+    u, created = kwargs["instance"], kwargs["created"]
+    if created:
+        permission = Permission.objects.get(codename='view_Case')
```

Comment:
Należy filtrować jeszcze według aplikacji dla której to uprawnienie jest, bo obecny zapis może być w przyszłości wieloznaczny, bo taką nazwę uprawnienia możemy mieć w kilku aplikacjach.